apps/explorer/src/components/HomeMetrics/index.tsx

```diff
@@ -12,9 +13,12 @@ import { Heading } from '~/ui/Heading';
 import { Stats, type StatsProps } from '~/ui/Stats';
 
 // Simple wrapper around stats to avoid text wrapping:
-function StatsWrapper(props: StatsProps) {
+function StatsWrapper({
+    isFirst,
+    ...props
+}: StatsProps & { isFirst?: boolean }) {
     return (
-        <div className="flex-shrink-0">
+        <div className={clsx('flex-shrink-0', isFirst && 'pl-8')}>
```

Comment:
Why can't this just get baked into the parent's CSS? Seems super weird to have this here